elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/Table.java

```diff
@@ -170,8 +169,8 @@ private boolean isFact(Class<?> cls, TableMeta meta) {
                 })
                 .collect(Collectors.toSet());
 
-        // add id field if exists
-        if (dictionary.getIdFieldName(cls) != null) {
+        // add id field if exists and this is not a fact model
+        if (!this.isFact() && dictionary.getIdFieldName(cls) != null) {
```

Comment:
Assuming if a model has `FromTable` annotation but isFact is set to false, we want `id` included in dimensions.